src/types/bytearray.rs

```diff
@@ -444,4 +456,125 @@ mod tests {
                 .is_instance_of::<PyValueError>(py));
         })
     }
+
+    // CPython 3.13t is unsound => test fails
+    #[cfg(any(Py_3_14, not(all(Py_3_13, Py_GIL_DISABLED))))]
```

Comment:
also needs to be skipped on emscripten

this failure mode happens often enough and wastes human time -  it's probably worth adding a linter checking that all tests using threads are disabled on platforms that don't support spawning threads.

Reply:
> it's probably worth adding a linter

Do you mean upstream in clippy or something else?